### DISCUSSION TITLE
Jekyll version of Github Page is too old

### DISCUSSION THREAD

**Discussion by YexuanXiao** [community/community#14535] [Other Feature Feedback, Questions, & Ideas]
I use Jekyll to build my personal blog site, but the GitHub Page version of Jekyll is too old, causing me to drop some features.

![image](https://user-images.githubusercontent.com/20025591/162385233-674681eb-e7c0-4bd0-83eb-9ad6313f32fe.png)

![image](https://user-images.githubusercontent.com/20025591/162385545-bec49a4c-b57b-43a6-98d2-cacf38ab4f2f.png)

Jekyll has provided many useful features since 4.0 and I wish there were more Jekyll versions to choose from rather than being stuck at 3.9.0.

**Comment by jfullstackdev** [comment]
did you install Jekyll manually ? 

there is this official site with the complete releases that you can select

https://jekyllrb.com/docs/history/
